extensions/HuggingFace/python/src/aiconfig_extension_hugging_face/remote_inference_client/text_generation.py

```diff
@@ -79,26 +82,25 @@ def construct_stream_output(
     accumulated_message = ""
     for iteration in response:
         metadata = {}
-        data = iteration
+        # If response_includes_details is false, `iteration` will be a string,
+        # otherwise, `iteration` is a TextGenerationStreamResponse
```

Comment:
Should we update `response` type to `Iterable[Union[TextGenerationStreamResponse, str]]` to reflect this?

Reply:
Good suggestion!